Sources/DistributedActors/Cluster/ClusterControl.swift

```diff
@@ -155,4 +155,77 @@ public struct ClusterControl {
     public func down(member: Cluster.Member) {
         self.ref.tell(.command(.downCommandMember(member)))
     }
+
+    public func joined(node: UniqueNode, within: Duration) async throws {
+        try await self.awaitMemberStatus(is: .up, within: within, node: node)
+    }
+
+    public func ensureNodes(_ status: Cluster.MemberStatus, within: Duration, nodes: [UniqueNode]) async throws {
+        try await withThrowingTaskGroup(of: Void.self) { group in
+            for node in nodes {
+                group.addTask {
+                    try await self.awaitMemberStatus(is: status, within: within, node: node)
+                }
+                try await group.next() // this is required to propagate error and cancel all tasks
+            }
+        }
+    }
+
+    public func ensureNodes(atLeast atLeastStatus: Cluster.MemberStatus, within: Duration, nodes: [UniqueNode]) async throws {
+        try await withThrowingTaskGroup(of: Void.self) { group in
+            for node in nodes {
+                group.addTask {
+                    try await self.awaitMemberStatus(atLeast: atLeastStatus, within: within, node: node)
+                }
+                try await group.next() // this is required to propagate error and cancel all tasks
+            }
+        }
+    }
+
+    public func awaitMemberStatus(atLeast atLeastStatus: Cluster.MemberStatus, within: Duration, node: UniqueNode) async throws {
```

Comment:
Naming seems inconsistent (maybe it was inconsistent in the testkit, sorry if so).

Let's try these shapes maybe:
- `try await system.cluster.waitFor(node, .up, within: ...)`
    - `try await system.cluster.waitFor(node, atLeast: .up, within: ...)`
- `try await system.cluster.waitFor([n1, n2, n2], .up, within: ...)`
    - `try await system.cluster.waitFor([n1, n2, n2], atLeast: .up, within: ...)`